Add generation of a boolean flange property to TriangulatedSets when extending with flange

emmanesbit opened the pull request.

andy-beer commented on resqpy/multiprocessing/wrappers/grid_surface_mp.py:
```diff
@@ -158,19 +172,31 @@ def find_faces_to_represent_surface_regular_wrapper(
+        flange_bool_property = Property.from_array(parent_model = model,
+                                                   cached_array = flange_bool,
+                                                   source_info = 'flange bool array',
+                                                   keyword = 'flange bool property',
```
let's drop 'property' from the title here; 'flange bool' or simply 'flange' is better I feel

emmanesbit commented:
@andy-beer i've updated the property variable as suggested, and added an additional unit test to check for the bool property. Could you please rerun yapf and merge when happy? :)
